_episodes_rmd/02-func-R.Rmd

```diff
@@ -7,6 +7,7 @@ questions:
 - "How can I test my functions?"
 - "How should I document my code?"
 objectives:
+- "Give the definition of the word function as used in programming"
```

Comment:
I don't think we need to add something theoretical here. How about we rather rephrase the next objective to "Make" or "Create …" so that we are reusing the wording of the `questions` above and avoid the duplicate meaning of `Define`?